### PR TITLE
FailoverClusters Bulk Fixup (2/4) - Update example formatting

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk updating example formatting before content updates

### CODE REVIEW

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 1]
```diff
@@ -37,58 +37,42 @@ authentication on the server computer.
...
-
-
-PS C:\> ForEach-Object -InputObject $groups -Process { $_ | Move-ClusterVirtualMachineRole -Node node2 }
-Name                       OwnerNode                           State 
-----                       ---------                           ----- 
-Virtual Machine1           node2                              Online 
-Virtual Machine2           node2                              Online 
-Virtual Machine3           node2                              Online
+```powershell
+$groups = Get-ClusterNode -Name node1 | Get-ClusterGroup | Where-Object -FilterScript {$_ | Get-ClusterResource | Where-Object -FilterScript {$_.ResourceType -Like "Virtual Machine"}}
```
Comment (robinharwood) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:68]: Hey @michaeltlombardi - I'm having a bad brain moment! Can you think of a nicer way to write this without rolling over the line?

Thanks
Robin

**Comment by robinharwood** [comment]
Hi @michaeltlombardi,

I believe this PR is release ready (minus on comment for you above) if you would be kind enough to review and if appropriate sign-off/merge 😊

Please let me know if you have any questions or if I need to apply any other updates.

Thanks Robin

File: docset/winserver2022-ps/failoverclusters/Get-ClusterResource.md [hunk 2]
```diff
@@ -34,131 +34,64 @@ and `Set-ClusterParameter`.
...
-the local cluster.
+This example lists cluster resources in cluster group named `FileServer1`, a clustered file server
+on the local cluster.
 
 ### Example 5
 
-```
-PS C:\> Get-ClusterResource -Name "Cluster Disk 2" | ForEach-Object -Process {$_.RestartDelay = 600}
+```powershell
+Get-ClusterResource -Name "Cluster Disk 2" | ForEach-Object -Process {$_.RestartDelay = 600}
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Get-ClusterResource.md:72]:
```suggestion
Get-ClusterResource -Name "Cluster Disk 2" | ForEach-Object -Process {
    $_.RestartDelay = 600
}
```

File: docset/winserver2022-ps/failoverclusters/Get-ClusterResource.md [hunk 3]
```diff
@@ -34,131 +34,64 @@ and `Set-ClusterParameter`.
...
-
-[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"):Y
+```powershell
+Get-ClusterResource -Name *print-VM1 | Get-VM | Stop-VM -Verbose -Confirm:$false
 ```
 
-This example enumerates the cluster resources for wildcard characters *print-VM1 and stops the
-corresponding virtual machines. Verbose mode is turned on for details of the operation.
+This example enumerates the cluster resources for wildcard characters `*print-VM1` and stops the
+corresponding virtual machines without user confirmation.. Verbose mode is turned on for details of
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Get-ClusterResource.md:93]:
```suggestion
corresponding virtual machines without user confirmation. Verbose mode is turned on for details of
```

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md [hunk 4]
```diff
@@ -37,58 +37,42 @@ authentication on the server computer.
...
+ForEach-Object -InputObject $groups -Process { $_ | Move-ClusterVirtualMachineRole -Node node2 }
 ```
 
 This example performs a live migration of all clustered virtual machines that are currently owned by
 the node named node1 to the node named node2. The migration of each virtual machine should complete
-before the next migration is started. Use this cmdlet before performing maintenance on the specified
-node.
+before the next migration is startqed. Use this cmdlet before performing maintenance on the
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:74]:
```suggestion
before the next migration is started. Use this cmdlet before performing maintenance on the
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
A couple minor fixes and the dangling comment thread.

One thing to keep in mind is that for longer lines of code that include pipes (`|`), you can break the line after the pipe and indent the following line. This generally makes pipelines easier to see and reason about.

For example,
This:
```powershell
Get-ClusterGroup -Name "Virtual Machine1" | Move-ClusterVirtualMachineRole -Node node2 -Wait 0
```
Can be written as:
```powershell
Get-ClusterGroup -Name "Virtual Machine1" |
    Move-ClusterVirtualMachineRole -Node node2 -Wait 0
```

This technique is particularly useful when you're doing 3+ piped commands together. At the commandline a user might just keep piping, but when reading the code in examples or scripts, the pipe-newline-indent pattern makes things more readable.

File: docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md (hunk 1 above)
Comment (michaeltlombardi) [reply docset/winserver2022-ps/failoverclusters/Move-ClusterVirtualMachineRole.md:68]: @robinharwood Happy with the PR aside from this last comment to resolve.

**Comment by robinharwood** [comment]
Hi @michaeltlombardi. I've updated example 4 in the Move-ClusterVirtualMachine command based on your brilliant advice. Please let me know if it needs an more tweaks otherwise ready for #sign-off

**Review by JohanFreelancer9 — APPROVED** [review]
Copy review.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**